pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -93,9 +112,10 @@ local function zotero_ref(cite)
 
     return pandoc.RawInline('openxml', field)
   else
+    local message = '<Do Zotero Refresh: ' .. content .. '>'
```

Comment:
This is still in the Zotero doc prefs, right? I prefer to have that here so people don't have to guess what "Zotero Refresh" means.

Reply:
> This is still in the Zotero doc prefs, right? I prefer to have that here so people don't have to guess what "Zotero Refresh" means.

Look at this screenshot https://github.com/retorquere/zotero-better-bibtex/pull/1592#discussion_r462259691(See Top Left on the Menu Bar).  'Set Document Preferences' is the gear wheel.  'Zotero Refresh' is the double arrow.  

When there no document preferences, clicking on Refresh will ask for preferences.    When there are document preferences, that is what ZOTERO_PREF_* in meta.xml does, then it will just go ahead without any prompts. 

If a user knows/has seen one, then surely you would have seen the other.

If you still insist that I restore the original line, I will.  Please confirm.